Rakefile

```diff
@@ -45,3 +45,65 @@ rule "gen/protoboeuf" => ->(_) { FileList["#{PROTO_PATH}/*.proto"] } do
 end
 
 task default: [:generate, :generate_protoboeuf]
+
+$benchmarks_run = 0
+def benchmark(name)
+  return if ENV["BENCH"] && !ENV["BENCH"].split(",").include?(name.to_s)
+
+  puts "\n\n" if ($benchmarks_run += 1) > 1
```

Comment:
Minor, but can we please avoid mutating values in a conditional? I'd rather see it as two different statements.